cmd/dagger/download.go

```diff
@@ -30,9 +30,9 @@ var downloadCmd = &FuncCommand{
 	BeforeRequest: func(_ *FuncCommand, cmd *cobra.Command, returnType *modTypeDef) error {
 		switch returnType.ObjectName() {
 		case Directory, File, Container:
-			flag := cmd.Flags().Lookup("export-path")
+			flag := cmd.Flags().Lookup("output")
 			if returnType.ObjectName() == Container && flag != nil && !flag.Changed {
-				return fmt.Errorf("flag --export-path is required for containers")
+				return fmt.Errorf("flag --output is required for containers")
```

Comment:
nit: doesn't this also apply for files/directories?

Reply:
The thing is that files and directories default to the current directory fine, so it's optional to set it. In containers you need to specify the path to a file, not a directory, that's why it's required.

Reply:
I think this is linked to @shykes's comment below - if we could just default all to a temp directory, that would be a lot clearer.